pinda-fun/pinda-fun
# Add instructions and countdown

### Changes
Added instructions screen and countdown screen before game starts, according to mockup.
Changed existing countdown as a common component for all games to use which manages its own state.

### How to test
Can go to [`/balloon-game`](https://deploy-preview-56--pinda-fun.netlify.com/balloon-game) to start the a balloon game to see it in action
